File: web-app/src/app/services/types/CampusCompany.ts
import { Campusregistration } from "./Campusregistration";
import { Companydatum } from "./Companydatum";

export type CampusCompany = {
  Id: number;
  CampusId?: number;
  CompanyId?: number;
  Campus: Campusregistration;
  Company: Companydatum;
};

export type PostCampusCompany = {
  Id: number;
  CampusId?: number;
  CompanyId?: number;
};

File: Placements.WebApi/Placement/Models/CampusCompany.cs
using System;
using System.Collections.Generic;

namespace Placements.WebApi.Placement.Models;

public partial class CampusCompany
{
    public int Id { get; set; }

    public long? CampusId { get; set; }

    public long? CompanyId { get; set; }

    public virtual Campusregistration? Campus { get; set; }

    public virtual Companydatum? Company { get; set; }
}
